apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryParameter.cs

```diff
@@ -212,12 +212,20 @@ public BigQueryParameter(string name, BigQueryDbType? type, object value)
             Value = value;
         }
 
+        /// <summary>
```

Comment:
I vaguely recall that this class was going to be renamed to "QueryParameter"?

Reply:
I was going to, yes - but there's a class called QueryParameter in the REST API as well, so it would clash with that. I figured it was cleaner to leave it as-is.

Reply:
`BigQueryParameter` is cleaner imho